--- origsrc/nautilus-image-converter-0.3.0/src/Makefile.am	2008-01-26 16:19:40.000000000 -0600
+++ src/nautilus-image-converter-0.3.0/src/Makefile.am	2008-06-04 22:52:39.515625000 -0500
@@ -25,7 +25,7 @@
 	nautilus-image-rotator.h		\
 	$(NULL)
 
-libnautilus_image_converter_la_LDFLAGS = -module -avoid-version
+libnautilus_image_converter_la_LDFLAGS = -module -avoid-version -no-undefined
 libnautilus_image_converter_la_LIBADD  = $(NAUTILUS_LIBS)
 
 dist_pkgdata_DATA = \
